### PR TITLE
Replace self.Owner with self:GetOwner() in TTT

### PR DESCRIPTION
**Pull request by roboderpy**

### CODE REVIEW

**Comment by svdm** [comment]
I was going to ask what the point of replacing it is, but then I noticed this is the first change I see in your PR:

    self:GetOwner() = data:GetEntity()

And I'm not going to waste any more time on this.

**Comment by roboderpy** [comment]
updated

File: garrysmod/gamemodes/terrortown/entities/effects/teleport_beamdown.lua [hunk 1]
```diff
@@ -8,7 +8,7 @@ local mid = 32
 local final_height = top
 
 function EFFECT:Init(data)
-   self.Owner = data:GetEntity()
+   self:GetOwner() = data:GetEntity()
```
Comment (ZehMatt) [inline garrysmod/gamemodes/terrortown/entities/effects/teleport_beamdown.lua]: ??

**Comment by Kefta** [comment]
@svdm self.Owner is deprecated. It only works because of an __index hack, and is slower.

**Pull request closed without merging**